README.md

```diff
@@ -2,19 +2,29 @@
 
 ## Description
 A mental health app for developers.
+The app allows users to search for videos about a mental health topic to help manage work-related stress.
 
 ## Installation
```

Comment:
What are the steps required to install your project? Provide a step-by-step description of how to get the development environment running.

(This project does not require any installation, so this section can just say "N/A")